### DISCUSSION TITLE
IP ACL / Block Internet Access

### DISCUSSION THREAD

**Discussion by mirisbowring** [goauthentik/authentik#3921] [General]
Hi all,

is it possible to attach a IP Blocklist / ACL to an application in order to prevent internet access?

I want to limit access to specific Applications only from local network.

Regards

**Comment by MichaelUray** [comment]
Did you actually figure that out?

**Reply by mirisbowring** [reply]
I realized this on my reverse proxy (nginx/caddy)

**Comment by rissson** [comment]
You can do it with an expression policy https://docs.goauthentik.io/docs/customize/policies/expression#comparing-ip-addresses and the `ak_client_ip` variable https://docs.goauthentik.io/docs/customize/policies/expression#variables
